assets/js/pages/homePage.js

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import HomePageContainer from 'containers/HomePageContainer';
+import ColorChanger from '../app/example-app';
 
-ReactDOM.render(<HomePageContainer />, document.getElementById('react-app'));
+
+const title = 'It really does work! (this section is rendered by React, change this message to test hot reloading)';
```

Comment:
`change this message to test hot reloading`. This does not work for me. Maybe this should be defined inside some component.